examples/shapes_and_collections/ellipse_demo.py

```diff
@@ -45,14 +45,14 @@
 delta = 45.0  # degrees
 
 angles = np.arange(0, 360 + delta, delta)
-ells = [Ellipse((1, 1), 4, 2, a) for a in angles]
+ellipses = [Ellipse((1, 1), 4, 2, a) for a in angles]
 
-a = plt.subplot(111, aspect='equal')
+ax = plt.subplot(111, aspect='equal')
 
-for e in ells:
-    e.set_clip_box(a.bbox)
-    e.set_alpha(0.1)
-    a.add_artist(e)
+for ellipse in ellipses:
```

Comment:
Seems vaguely nicer to loop over angles and create one ellipse at a time in the loop.